INSTALL.rst

```diff
@@ -39,12 +54,9 @@ The HaaS software can then be installed by running:
 Disable SELinux
 ---------------
 
-The setup described below runs into problems with SELinux related to the sqlite
-database. For now the recommended solution is to simply disable SELinux. In
-future releases, we will support and recommend the use of SELinux with another
-DBMS, such as MySQL.
-
-::
+The setup described in this document runs into problems with SELinux. In the
+future, we hope to ship a set of SELinux security policies with HaaS, but for
+now the solution is to disable SELinux::
```

Comment:
If disabling SELinux was for sqlite, do deployers still need to disable it for Postgres?